### PR TITLE
[Dubbo-4871] fix cache file name issue in windows platform

### PR DESCRIPTION
**Pull request by JoeKerouac**
Add Windows support, corresponding issue:
[在windows连接zookeeper出现文件写入异常](https://github.com/apache/dubbo/issues/4871)

### CODE REVIEW

**Comment by JoeKerouac** [comment]
对应的issue:#4871

**Comment by JoeKerouac** [comment]
@htynkn 有空帮忙review下吧

File: dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java [hunk 1]
```diff
@@ -1,511 +1,512 @@
...
-        }
-
-        @Override
-        public void run() {
-            doSaveProperties(version);
-        }
-    }
-
-}
+/*
```
Comment (htynkn) [inline dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java]: What changes in this file? If no changes for this file, please revert it.

Comment (JoeKerouac) [reply dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/support/AbstractRegistry.java]: I only modified 95 and 96 lines, the local comparison is only modified 95 and 96 lines, I don't know why the whole file is modified here.

**Comment by htynkn** [comment]
Thanks very much for your contribution.

**Pull request merged**